BuildResidentialScheduleFile/measure.rb

```diff
@@ -56,6 +56,16 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDescription('Specifies the vacancy period. Enter a date like "Dec 15 - Jan 15".')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('schedules_outage_period', false)
+    arg.setDisplayName('Schedules: Outage Period')
+    arg.setDescription('Specifies the outage period. Enter a date/time like "Dec 15 10am - Jan 15 2pm". 12am is the final hour of a given day.')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeBoolArgument('schedules_outage_window_natvent_availability', false)
+    arg.setDisplayName('Schedules: Outage Period Natural Ventilation Availability')
+    arg.setDescription('Whether natural ventilation is available during the outage period.')
+    args << arg
```

Comment:
@shorowit Would it be more appropriate to generate the `natural_ventilation_availability` schedule from `ResStockArgumentsPostHPXML`? Or do you think it's useful to have this feature work inside `BuildResidentialScheduleFile`?

Reply:
I haven't thought through all the implications, but my initial reaction is to do it in `BuildResidentialScheduleFile` because it seems generally useful for non-ResStock users too.

Reply:
To be clear, I was talking about only generating the `natural_ventilation_availability` schedule when an outage period is specified. In the future (see note above), we could think about generating the schedule in a way that is consistent with occupant schedules, etc.